metaflow/plugins/resources_decorator.py

```diff
@@ -49,7 +49,11 @@ def myStep(self):
         Not for AWS Batch.
         KFP: Memory limit for this step. Default unit is MB.
             More memory units are supported, including "E", "P", "T", "G", "M", "K". (i.e. "4000M")
```

Comment:
Minor: I would also add here the `Mi`, `Gi`, etc resource settings are also available.

We might even want to suggest these units because the kubeflow-k8s-profiles uses these resource settings for the quotas.

Reply:
Sure. I haven't change this part but it's a good suggestion.